plugins/synaptics-mst/fu-synaptics-mst-device.c

```diff
@@ -771,19 +772,133 @@ fu_synaptics_mst_device_panamera_prepare_write (FuSynapticsMstDevice *self, GErr
 	return TRUE;
 }
 
+static gboolean
+fu_synaptics_mst_device_update_cayenne_firmware (FuSynapticsMstDevice *self,
+						    guint32 payload_len,
+						    const guint8 *payload_data,
+						    GError **error)
+{
+	g_autoptr(FuSynapticsMstConnection) connection = NULL;
+	guint8 bytedata[4];
+	gboolean mapping_enabled = FALSE;
```

Comment:
both of these variables are not used, and CI fails from them

Reply:
Hi superm1,
Many thanks for your reviews.
Now I have done the modifications based on your reviews and suggestions.
I have git push the commit to https://github.com/apolloling/fwupd
What do I need to continue the PR ?